src/microbe_stage/editor/MicrobeEditor.cs

```diff
@@ -1884,11 +1884,10 @@ private void UndoOrganellePlaceAction(MicrobeEditorAction action)
 
     private bool AddOrganelle(OrganelleTemplate organelle)
     {
-        // 1 - you put nucleus but you already have it
-        // 2 - you put organelle that need nucleus and you don't have it
+        // 1 - you put a unique organelle (means only one instance allowed) but you already have it
+        // 2 - you put organelle that requires nucleus but you don't have one
```

Comment:
```suggestion
        // 2 - you put an organelle that requires nucleus but you don't have one
```
Just determiner consistency, but the comment change is nice already